packages/mds-api-helpers/package.json

```diff
@@ -14,14 +14,18 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
-    "test": "DOTENV_CONFIG_PATH=../../.env nyc --use-spawn-wrap --lines 40  ts-mocha --project ../../tsconfig.json"
+    "test": "DOTENV_CONFIG_PATH=../../.env jest --runInBand --forceExit "
```

Comment:
Do we need the `forceExit` here?

Reply:
Pretty sure we don't, as russen got rid of it in his PR